src/main/java/qna/domain/DeleteHistories.java

```diff
@@ -0,0 +1,29 @@
+package qna.domain;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+public class DeleteHistories {
+
+    List<DeleteHistory> deleteHistories;
+
+    public DeleteHistories () {
+        deleteHistories = new ArrayList<>();
+    }
+
+    public void add(DeleteHistory deleteHistory) {
+        deleteHistories.add(deleteHistory);
+    }
+
+    public void addAll(DeleteHistories deleteHistories) {
+        for (DeleteHistory deleteHistory : deleteHistories.getDeletedHistories()) {
+            this.deleteHistories.add(deleteHistory);
+        }
```

Comment:
기존에 있는 메소드 활용해주셔도 좋을 것 같아요 😆
```suggestion
        this.deleteHistories.addAll(deleteHistories.getDeletedHistories());
```